.github/workflows/tag.yml

```diff
@@ -32,7 +32,7 @@ env:
 jobs:
   tag:
     name: Tag
-    runs-on: self-hosted
+    runs-on: self-hosted-x64
```

Comment:
Should such a light workflow also use `ubuntu-latest` or what do we make that dependent on?

Reply:
Yeah, I did not even understand why we had this `Tag` workflow tbh. Imho it can be removed.

Reply:
Seems to abuse the tags to create release notes if release_notes_monitor.yml would be enabled

Reply:
Can be handled as part of https://github.com/iotaledger/iota/issues/4347 I guess

Reply:
But do you want to change stale.yml to use `ubuntu-latest`?